<?xml version="1.0"?>
<launch>
  <arg name="log_type" default="screen"/>

<!--   <node pkg="kinova_unit_app" type="object_detect_node" name="object_detect_node" output="$(arg log_type)">
    <param name="point_cloud_frame" value="kinect2_rgb_optical_frame" /> 
    <param name="point_cloud_topic" value="kinect2/qhd/points" />
    <param name="debug_output_topic" value="/debug_pointcloud_output" />
    <param name="object_visual_markers_topic" value="/object_visual_markers" />
  </node> -->

  <!-- <node pkg="kinova_unit_app" type="kinova_unit_app_node.py" name="kinova_unit_app_node" output="$(arg log_type)"> -->
  <!-- </node> -->
  <node pkg="kinova_unit_app" type="object_recognition" name="object_detect_node" output="$(arg log_type)">
    <param name="point_cloud_frame" value="kinect2_rgb_optical_frame" /> 
    <param name="point_cloud_topic" value="kinect2/hd/points" />
    <param name="debug_output_topic" value="/debug_pointcloud_output" />
    <param name="object_visual_markers_topic" value="/object_visual_markers" />
  </node>
</launch>
